src/commands/synthetics/__tests__/cli.test.ts

```diff
@@ -153,127 +186,119 @@ describe('run-test', () => {
 
       const command = new RunTestCommand()
       command['apiKey'] = 'api_key_cli'
+      command['mobileApplicationVersionFilePath'] = './path/to/application_cli.apk'
 
       await command['resolveConfig']()
       expect(command['config']).toEqual({
         ...DEFAULT_COMMAND_CONFIG,
         apiKey: 'api_key_cli',
         appKey: 'app_key_env',
         datadogSite: 'us5.datadoghq.com',
-        global: {pollingTimeout: DEFAULT_POLLING_TIMEOUT},
-      })
-    })
-
-    test('pass command pollingTimeout as global override if undefined', async () => {
-      const command = new RunTestCommand()
-      command.configPath = 'src/commands/synthetics/__tests__/config-fixtures/config-with-global-polling-timeout.json'
-      await command['resolveConfig']()
-      expect(command['config']).toEqual({
-        ...DEFAULT_COMMAND_CONFIG,
-        configPath: 'src/commands/synthetics/__tests__/config-fixtures/config-with-global-polling-timeout.json',
-        global: {followRedirects: false, pollingTimeout: 333},
-        pollingTimeout: 333,
+        global: {
+          pollingTimeout: DEFAULT_POLLING_TIMEOUT,
+          mobileApplicationVersionFilePath: './path/to/application_cli.apk',
+        },
       })
     })
 
-    test('override locations with ENV variable', async () => {
-      const conf = {
-        content: {tests: [{config: {}, id: 'publicId'}]},
-        name: 'Suite 1',
-      }
-
-      jest.spyOn(ciUtils, 'resolveConfigFromFile').mockImplementation(async (config, _) => config)
-      jest.spyOn(utils, 'getSuites').mockImplementation((() => [conf]) as any)
-
-      // Throw to stop the test
-      const triggerTests = jest.fn(() => {
-        throw getAxiosHttpError(502, {message: 'Bad Gateway'})
```

Comment:
My understanding is that this was used to prevent actually triggering the tests. What's implemented to prevent triggering the tests? Switching from `triggerTests` to `getTestsToTrigger`?